### PR TITLE
Update readme with migration notice to CCI

### PR DESCRIPTION
**Pull request by Croydon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,8 @@
-# Conan libraries and code wishlist
+# Go to [Conan Center Index instead](https://github.com/conan-io/conan-center-index)
```
Comment (uilianries) [inline README.md]:
```suggestion
## Conan libraries and code wishlist

# DEPRECATION NOTICE

If you want to request a new package, please go to [Conan Center Index](https://github.com/conan-io/conan-center-index) instead.
```

File: README.md [hunk 2]
```diff
@@ -1,68 +1,8 @@
...
 
-This repository is a community effort used to track requests for code and packages that users would like to have available
-in conan. This Readme might contain a **summary** of the requests and status, as well as recent news. But the
-main flow of information is via github issues, please read the following guide:
+This repository tracked user wishes for new Conan recipes prior to the start of the [Conan Center Index](https://github.com/conan-io/conan-center-index).
 
+Please go there now. If a Conan package does not exist in the Index yet, please search the issues first, if someone else already made a request for the library or tool you would like to see. If such a request does not exist, feel free to create one.
```
Comment (uilianries) [inline README.md]:
```suggestion
- If a Conan package does not exist in the Index yet, please search the issues first;
- If someone else already made a request for the library or tool you would like to see;
- If such a request does not exist, feel free to [create](https://github.com/conan-io/conan-center-index/issues/new?labels=library+request&template=package_request.md&title=%5Brequest%5D+%3CLIBRARY-NAME%3E%2F%3CLIBRARY-VERSION%3E) one.
```

Comment (Croydon) [reply README.md]: Actually, that semicolon makes no sense there 🤔

Comment (uilianries) [reply README.md]: sorry! it's Portuguese grammar :sweat_smile: It was automatic

File: README.md [hunk 3]
```diff
@@ -1,68 +1,11 @@
-# Conan libraries and code wishlist
+# Conan Wishlist
 
-This repository is a community effort used to track requests for code and packages that users would like to have available
-in conan. This Readme might contain a **summary** of the requests and status, as well as recent news. But the
-main flow of information is via github issues, please read the following guide:
+## DEPRECATION NOTICE: Go to [Conan Center Index instead](https://github.com/conan-io/conan-center-index)
```
Comment (danimtb) [inline README.md]:
```suggestion
## DEPRECATION NOTICE: Use  [Conan Center Index repository](https://github.com/conan-io/conan-center-index) instead.
```

**Review by danimtb — APPROVED** [review]
Minor comments

With this update we can disable the creation of new issues in this repo

**Comment by danimtb** [comment]
Maybe we have to wait to close the issue tracker until we are out of the EAP

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**